Examples/Genrefy/Genrefy/Activities/Analysis/AnalysisCoordinator.swift

```diff
@@ -49,7 +49,19 @@ class AnalysisCoordinator: NSObject, ActivityCoordinating {
 
     func begin() {
         let accountsVC = AccountsViewController.instantiate()
-        let dataSource = AccountSelectionDataSource(accounts: repository?.accounts ?? [])
+        var accounts = repository?.accounts ?? []
+    
+        if
+            accounts.isEmpty,
+            let persistentData = PersistentStorage.shared.loadData(for: "accounts.json"),
+            let persistentDic = try? NSKeyedUnarchiver.unarchiveTopLevelObjectWithData(persistentData) as? [AnyHashable: Any],
```

Comment:
The dictionary is effectively serialized `persisted` data, therefore should be renamed to `persistedDict`, note I also added a `t` on the end there :)